src/gui/rss/rsswidget.cpp

```diff
@@ -461,10 +462,10 @@ void RSSWidget::handleCurrentArticleItemChanged(QListWidgetItem *currentItem, QL
     }
     else {
         QString description = article->description();
-        QRegExp rx;
+        QRegularExpression rx;
```

Comment:
```c++
const QRegularExpression rx {"\\[img\\](.+)\\[/img\\]", 
QRegularExpression::InvertedGreedinessOption | QRegularExpression::CaseInsensitiveOption};
```
???

Reply:
it can't be const, there are a series of setPattern() below.